adafruit_touchscreen.py

```diff
@@ -104,7 +104,8 @@ def __init__(
         samples: int = 4,
         z_threshold: int = 10000,
         calibration: Optional[Tuple[Tuple[int, int], Tuple[int, int]]] = None,
-        size: Optional[Tuple[int, int]] = None
+        size: Optional[Tuple[int, int]] = None,
+        invert_pressure: bool = True
```

Comment:
Can you please document this in the docstring above?  Thanks!

Reply:
docstring has been added in the latest commit.